src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -440,7 +440,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_degrees)
 
 // radians
 MXNET_OPERATOR_REGISTER_UNARY(radians)
-.describe(R"code(Convert angles from degrees to radians.
+.describe(R"code(Returns element-wise angles in radians converted from degree angles of the input.
```

Comment:
Same issues. Just write:

"Converts each element of the input array from degrees to radians."